compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2375,18 +2375,16 @@ setLoweringConfigForComputeOps(mlir::FunctionOpInterface entryPointFn,
                        << "\n");
 
   // Split parallel vector tile sizes into common parts and op-specific parts.
-  SmallVector<int64_t> commonVecTileSizes = parallelVecTileSizes;
-  SmallVector<bool> commonVecScalableTileFlags = parallelVecScalableTileSizes;
-  SmallVector<int64_t> innerVecTileSizes(maxLoopNums, 0);
-  SmallVector<bool> innerVecScalableTileFlags(maxLoopNums, false);
+  SizesAndScalableFlagsTuple commanVecTileSizesAndFlags = {
+      parallelVecTileSizes, parallelVecScalableTileSizes};
```

Comment:
I think we can directly use the constructor here? E.g.,

```suggestion
  SizesAndScalableFlagsTuple commanVecTileSizesAndFlags(parallelVecTileSizes, parallelVecScalableTileSizes);
```